assets/src/edit-story/components/canvas/pageAttachment/index.js

```diff
@@ -0,0 +1,175 @@
+
+const Wrapper = styled.div`
+  position: absolute;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
+  bottom: 0;
+  height: 20%;
+  width: 100%;
+  color: ${({ theme }) => theme.colors.fg.v1};
+  z-index: 3;
+`;
+
+const Guideline = styled.div`
+  mix-blend-mode: difference;
+  position: absolute;
+  height: 1px;
+  bottom: 20%;
+  width: 100%;
+  background-image: ${({ theme }) =>
+    `linear-gradient(to right, ${theme.colors.fg.v0} 50%, ${theme.colors.fg.v1} 0%)`};
+  background-position: top;
+  background-size: 16px 0.5px;
+  background-repeat: repeat-x;
+  z-index: 3;
+`;
+
+// The CSS here is based on how it's displayed in the front-end, including static
+// font-size, line-height, etc. independent of the viewport size -- it's not responsive.
+const Icon = styled.div`
+  position: relative;
+  display: block;
+  height: 28px;
+  width: 32px;
+`;
+
+const arrowBarCss = css`
+  position: absolute;
+  display: block;
+  height: 3px;
+  width: 12px;
+  border-radius: 3px;
+  top: 14px;
+  background: ${({ theme }) => theme.colors.fg.v1};
+  box-shadow: 0px 0px 6px rgba(0, 0, 0, 0.36);
+`;
+
+const LeftBar = styled.div`
+  ${arrowBarCss}
+  left: 6px;
+  transform: rotate(-30deg);
+`;
+
+const RightBar = styled.div`
+  ${arrowBarCss}
+  right: 6px;
+  transform: rotate(30deg);
+`;
+
+const TextWrapper = styled.span`
+  font-family: Roboto;
+  font-size: 16px;
+  line-height: 16px;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  text-shadow: 0px 0px 6px rgba(0, 0, 0, 0.36);
+  white-space: nowrap;
+  max-width: 90%;
+  position: relative;
+  padding: 0 32px;
+  height: 16px;
+  letter-spacing: 0.3px;
+`;
+
+const Tooltip = styled.div`
+  background-color: ${({ theme }) => theme.colors.bg.v0};
+  color: ${({ theme }) => theme.colors.fg.v1};
+  width: 200px;
+  padding: 8px;
+  font-size: 14px;
+  border-radius: 4px;
+  text-align: center;
+`;
+
+const spacing = { x: 8 };
+
+function PageAttachment({ pageAttachment = {} }) {
+  const {
+    displayLinkGuidelines,
+    pageAttachmentContainer,
+    setPageAttachmentContainer,
+  } = useCanvas((state) => ({
+    displayLinkGuidelines: state.state.displayLinkGuidelines,
+    pageAttachmentContainer: state.state.pageAttachmentContainer,
+    setPageAttachmentContainer: state.actions.setPageAttachmentContainer,
+  }));
+
+  const { hasInvalidLinkSelected } = useElementsWithLinks();
+
+  const { ctaText = __('Learn more', 'web-stories'), url } = pageAttachment;
+  return (
+    <>
+      {(displayLinkGuidelines || hasInvalidLinkSelected) && <Guideline />}
+      <Wrapper role="presentation" ref={setPageAttachmentContainer}>
+        {url?.length > 0 && (
+          <>
+            <Icon>
+              <LeftBar />
+              <RightBar />
+            </Icon>
+            <TextWrapper>{ctaText}</TextWrapper>
+            {pageAttachmentContainer && hasInvalidLinkSelected && (
```

Comment:
So this is a bit simple and not super informative while dragging something, but I think it's okay for now until we get the "real" pre-publish checklist and the proper way of displaying these errors.

However, I have a few issues with this message:

![Screen Shot 2020-08-14 at 14 40 57](https://user-images.githubusercontent.com/637548/90282429-52fdf300-de3c-11ea-87ca-99638d62fa4f.png)

1. Where's the dashed line?
2. Placement is... aggressive!

Reply:
Thanks for testing!

> So this is a bit simple and not super informative while dragging something, but I think it's okay for now until we get the "real" pre-publish checklist and the proper way of displaying these errors.

Originally it was reporting errors while dragging as well but this is what we agreed on after the logic got too complicated and all-over-the-place handling displaying the errors in multiple places since every place needed a bit different handling. Having one pre-publish checklist to pull information from will hopefully fix this indeed :)

> Placement is... aggressive!

Is this with a smaller screen? I suppose that needs some UX attention then. Will check.